doc/doxygen/images/pieces.svg

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is this file used somewhere in the documentation?

Reply:
This file will be used in the future.